sites/partners/cypress/integration/07-jurisdictional-admin-user-management.spec.ts

```diff
@@ -1,13 +1,16 @@
-describe("Admin User Mangement Tests", () => {
-  before(() => {
-    cy.loginAndAcceptTerms("jurisdictionalAdmin")
+describe("Jurisdictional Admin User Mangement Tests", () => {
+  beforeEach(() => {
+    cy.login("jurisdictionalAdmin")
   })
 
-  after(() => {
+  afterEach(() => {
     cy.signOut()
   })
 
   it("as jurisdictional admin user, should only see partners/jurisdictional admins on the same jurisdiction", () => {
+    cy.getByTestId("agree").check()
```

Comment:
This makes it so that the tests below expect this test to run first since the first test clicks the agree button and all following tests assume you have signed into an account that has agreed to the terms. This is fine for this PR, but another way to get around this is to make the "jurisdictionalAdmin" be agreed by default when the data is seeded.

Reply:
this turned out to be a bug in the way the data was seeded that I didn't catch. 

Got this updated